server/routes.go

```diff
@@ -97,6 +97,13 @@ func load(c *gin.Context, model *Model, opts api.Options, sessionDuration time.D
 		loaded.Options = &opts
 	}
 
+	if sessionDuration < 0 {
```

Comment:
This isn't necessary. sessionDuration (keepAlive) is explicitly set to math.MaxFloat64 if it's less than 0

Reply:
That actually would introduce a bug because the duration can be set as either a float64, or as a string value. If it was set as "-1s" then it would automatically unload the model after generate.

Reply:
IMO this is the wrong place for this. A negative value for the `keep_alive` field should be explicitly interpreted as `keep indefinitely` when unmarshalled rather this deep in the handler.

Since unmarshalling already handles negative float durations, it should ideally handle negative string durations

Reply:
I actually ended up refactoring this to make parsed values work the same way.